src/HaroohiePals.MarioKartToolbox/KCollision/MkdsKclPrismAttribute.cs

```diff
@@ -68,6 +68,11 @@ public bool IsFloor
         set => SetBits(value ? 1 : 0, 15, 1);
     }
 
+    public bool UnusedFlag
+    {
+        get => GetBits(0, 1) == 1;
+        set => SetBits(value ? 1 : 0, 0, 1);
+    }
```

Comment:
Add another line break
```suggestion
    }
    
```